### PR TITLE
For #12759 - Keep a separate list of release locales.

### PR DESCRIPTION
**Pull request by person808**
This also updates the `release_locale` list to match the locales on the`releases/v79.0.0` branch. Tested the filtering script locally and it worked for me.

cc @liuche Do we want to uplift this to the release branch?

### CODE REVIEW

**Comment by liuche** [comment]
@person808 this doesn't need to be uplifted - pontoon only localizes from master, so changing the list on the release branch doesn't have any effect, beyond removing those locale dirs that we don't want to include.

File: automation/taskcluster/l10n/locales.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def get_release_locales():
     for line in file:
         if line == OPEN_LOCALES:
             locales_opened = True
-        elif line == CLOSE_LOCALES:
+        elif line == CLOSE_LOCALES and locales_opened == True:
```
Comment (liuche) [inline automation/taskcluster/l10n/locales.py:45]: Does this catch an edge case?

Comment (person808) [reply automation/taskcluster/l10n/locales.py:45]: It's so we don't exit the loop when we see the closing bracket for the regular `locales` list.

**Comment by person808** [comment]
Hm. Just realized that the locale list I tested the TC apk with had all the locales in `l10n.toml` so there would have been no difference in locales even with the script running. It should still be working (https://github.com/mozilla-mobile/fenix/issues/3088#issuecomment-592170944) but I'll merge this tonight so I can double check a TC apk tomorrow.

**Comment by Pike** [comment]
This changes an l10n project  configuration file w/out any feedback or review from the l10n team.

This is not OK, and I expect additional entries in l10n.toml files to be errors in linters at some point.

Please find a different spot to put this data.

CC @Delphine

**Pull request merged**